app/templates/components/post-header.hbs

```diff
@@ -1,2 +1,2 @@
 <div class="post-icon"></div>
-{{post-title post=post}}
+{{post-title post=post saveTitle=saveTitle}}
```

Comment:
The only problem I can see is getting confused about which `save` function is being used. The route's function is called `save`, the component's function is called `save`, and now we're going to call the route's passed down function `save`.